R/style_guides.R

```diff
@@ -1,11 +1,26 @@
+#' Style guides
+#'
+#' Format code according to a style guide. Style guides are the input to the
+#' argument `style` in [style_file()] and friends.
+#' The available style guides are:
+#' * the tidyverse style guide (see [tidyverse_style()]).
+#' @name style_guides
+NULL
+
+
 #' The tidyverse style
+#'
+#' Style code according to the tidyverse style guide.
 #' @param scope The extent of manipulation. Can range from "none" (least
 #'   invasive) to "token" (most invasive). See 'Details'. This argument is a
 #'   vector of length one.
 #' @param indent_by How many spaces of indention should be inserted after
 #'   operators such as '('.
 #' @param strict A logical value indicating whether a set of strict
-#'   or not so strict transformer functions should be returned.
+#'   or not so strict transformer functions should be returned. `strict = TRUE`
```

Comment:
We might want to assign more semantics to the `strict` argument later on, maybe describe these only as examples what `strict` can affect?